src/foam/doc/DigSelectView.js

```diff
@@ -0,0 +1,134 @@
+foam.CLASS({
+  package: 'foam.doc',
+  name: 'DigSelectView',
+  extends: 'foam.u2.View',
+  implements: [
+    'foam.mlang.Expressions',
+  ],
+  imports: [
+    'appConfig',
+    'user',
+  ],
+  requires: [
+    'foam.doc.CodeTabs',
+    'foam.u2.Tab',
+    'foam.doc.dao.AxiomDAO',
+    'foam.doc.PropertyAxiom',
+    'foam.doc.CodeTab',
+    'foam.flow.PromiseSlot',
+    'foam.u2.DetailView',
+    'foam.nanos.dig.DIG',
+  ],
+  properties: [
+    {
+      class: 'StringArray',
+      name: 'digProperties',
+      factory: function() {
+        return [
+          'daoKey',
+          'format',
+          'q',
+        ];
+      },
+    },
+    {
+      name: 'url',
+      expression: function(appConfig) {
+        if ( appConfig ) return appConfig.url;
+      }
+    },
+    {
+      name: 'samplequery',
+      experssion: function(appConfig) {
+        if ( appConfig ) return appConfig.q;
```

Comment:
appConfig.q isn't a thing. This is the AppConfig model:
https://github.com/foam-framework/foam2/blob/master/src/foam/nanos/app/AppConfig.js